packages/synapse-react-client/src/components/Forum/DiscussionSearchResult.tsx

```diff
@@ -78,23 +79,23 @@ const DiscussionSearchResult = (props: DiscussionSearchResultProps) => {
   }
 
   return (
-    <div className="bootstrap-4-backport search-result-container">
-      <Row>
-        <Col xs={1}>
+    <div className="search-result-container">
+      <Box display={'grid'} gridTemplateColumns={'40px auto'}>
```

Comment:
The left column just shows an icon indicating if this result is the root of the thread or a reply. It looks better with a static 40px width instead of 1/12 of the total width, so use CSS grid